libr/core/cmd_debug.c

```diff
@@ -1570,15 +1570,37 @@ static void cmd_debug_reg(RCore *core, const char *str) {
 			r_core_cmd_help (core, help_message);
 		}
 		break;
-	case 'l': // "drl"
-		//r_core_cmd0 (core, "drp~[1]");
+	case 'l': // "drl[j]"
 		{
-			RRegSet *rs = r_reg_regset_get (core->dbg->reg, R_REG_TYPE_GPR);
-			if (rs) {
-				RRegItem *r;
-				RListIter *iter;
-				r_list_foreach (rs->regs, iter, r) {
-					r_cons_println (r->name);
+			bool json_out = false;
+			switch (str[1]) {
+				case 'j':
+					json_out = true;
```

Comment:
if there's no break add a /* fall thru */ comment . this will silent coverity and make clear that its not a mistake